.github/workflows/main.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - run: yarn lint
       - run: yarn test
       - run: yarn build || { echo "build command should always succeed" ; exit 61; }
-      - run: yarn build --quiet && git diff --quiet action || { echo "action should be auto generated" ; exit 62; }
+#      - run: yarn build --quiet && git diff --quiet dist || { echo "dist should be auto generated" ; exit 62; }
```

Comment:
Should we remove that line?

Reply:
Prefer to keep it for now, since it's still in unity-builder and we may want to add back in a line that's similar.
I think we can update it or remove it from all 4 (`unity-request-activation-file`, `unity-activate`, `unity-test-runner`, and `unity-builder`) at the same time later.